src/applications/personalization/rated-disabilities/tests/components/RatedDisabilityView.unit.spec.jsx

```diff
@@ -8,7 +8,9 @@ describe('<RatedDisabilityView/>', () => {
   const user = {
     profile: {
       verified: true,
-      status: 'OK',
```

Comment:
this one should stay as is; `profile.status` is still a thing and different fron `profile.veteranStatus.status`

Reply:
You're right, good catch! Updated 👩‍💻